src/main/java/uk/gov/hmcts/reform/idam/health/CustomRequestRetryer.java

```diff
@@ -0,0 +1,38 @@
+package uk.gov.hmcts.reform.idam.health;
+
+import feign.RetryableException;
+import feign.Retryer;
+
+public class CustomRequestRetryer implements Retryer {
```

Comment:
Might need to talk about this - we don't want retry on the feign calls (it would affect the health check), and I'm not expecting the credentials stuff to use feign. Haven't had a chance to look through all the code yet though.